status-im/status-react
# [bug] fix #2128 No feedback on insufficient funds from unsigned

fixes #2128 
fixes #2054
fixes #2132 (by merging PR)

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
- move current unsigned-transaction to wallet
- move some logic from view to subscriptions and events

<img width=320 src=https://user-images.githubusercontent.com/1181225/31547509-e6e00874-b027-11e7-930d-be5273844774.png />

<img width=320 src=https://user-images.githubusercontent.com/1181225/31551843-15127e9e-b036-11e7-90e7-7d70775231f4.png />

### Steps to test:
- Open Status
- Create transaction with insufficient funds and sign later
- Sign transaction from unsigned and check that the Sign transaction action is disabled and warning about insufficient funds is shown

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready
